### DISCUSSION TITLE
Can´t login after install

### DISCUSSION THREAD

**Discussion by marceloasuarez** [givanz/VvvebJs#319] [Q&A]
Hi,

Congratulations for your great work, simply excellent!
I have installed from scratch 3 times already, but it does not accept my login and password immediately after finishing the installation. I don't know what I'm doing wrong?

**Comment by givanz** [comment]
Hi

Thank you.

Can you please provide more details about your setup? like php and database.

I couldn't reproduce on php 8.2.8 and MariaDB 10.6.12.

Does it work with user:`admin` password:`admin`? these are the defaults before they get overwritten by the ones provided on install.

If possible you can send me temporary ssh/ftp etc access to your setup and I will debug the issue to find the cause.

You can use the email address on my profile page or the contact form http://www.vvveb.com/page/contact

Thanks.

**Comment by marceloasuarez** [comment]
Hello Givanz,
Thank you very much for your quick response.

I hired Hostinger as a server, the default PHP version is 8.2.

The configuration that appears in my Phpmyadmin is:
Database version: libmysql - mysqlnd 8.1.16
PHP version: 8.1.16

I installed it in a subdirectory, so I made modifications in the index.php
files that are in the main directory, install and admin directory, to
indicate to the system that I would install it through the directory:
vvvebjscms.

To access the system is through:
https://marcelosuarez.com.br/vvvebjscms/admin/
The User is: admin, and the password: ij4Dmhz$OI4aH9x9#^$^L)G^.

To access via FTP :
ftp.marcelosuarez.com.br
***@***.***
senha=Vvvebjs@

Yours sincerely,

**Comment by givanz** [comment]
Github masks email addresses and FTP details are not visible, can you please send the authentication details to the email address from my github profile?

Thanks

**Reply by marceloasuarez** [reply]
Hi Givanz,

Here the details to access to the site by FTP:

[ftp.marcelosuarez.com.br](http://ftp.marcelosuarez.com.br/)
user=[givanz@marcelosuarez.com.br](mailto:givanz@marcelosuarez.com.br)
pass=Vvvebjs@

To access the system is through: https://marcelosuarez.com.br/vvvebjscms/admin/
The User is: admin, and the password: ij4Dmhz$OI4aH9x9#^$^L)G^.

Thank in advance,

Marcelo.

**Comment by marceloasuarez** [comment]
OK, I send it by github.

Thank in advance,

**Comment by givanz** [comment]
Thanks for access, admin login issue was caused by php session cookie not working properly because of subfolder install.

The session cookie is set for all subdomains for multisite websites to share the same session and this made the session not to work at all.

I disabled subdomain session sharing when installed in subfolder and now login works.

Subdomain install is not thoroughly tested if you find other bugs let me know and I will fix them.

I think mod rewrite is not enabled on your apache server because links like https://marcelosuarez.com.br/vvvebjscms/blog don't work.

https://marcelosuarez.com.br/vvvebjscms/admin/
admin 
ij4Dmhz$OI4aH9x9#^$^L)G^.

Please change your admin password and disable FTP access.

Best regards.

**Comment by marceloasuarez** [comment]
Hi,

Let me see if I understand, to make an installation I enable the subdomain
session, and once the session is finished I disable it, is that right?

Best Regards,

**Reply by givanz** [reply]
Hi

You don't have to do anything, if `V_SUBDIR_INSTALL` is defined then shared session is disabled to prevent issues.
Shared subdomain session for subdir installation doesn't make sense anyway and it should have been disabled from the beginning.

**Comment by marceloasuarez** [comment]
Hi Givanz,
I found 2 inconsistencies in the system for you to correct in
futurse versions:
1) When I logged in, it offered me to update, I did it, but after finishing
the update the system lost the styles and would not let me access it again.
I solved it by adding this line in the index.php of the root and admin
directory: define('V_SUBDIR_INSTALL', '/vvvebjscms');
2) When I put the Theme in dark mode, and go to another section, it goes
back to light mode only.

Best Regards,

Marcelo.

**Comment by givanz** [comment]
Hi Marcelo,

Sorry for this, the update didn't skip the env.php file where `V_SUBDIR_INSTALL` is defined, I changed the code to skip this file on update.

You can keep `V_SUBDIR_INSTALL` in env.php to avoid losing the change on future updates.

The dark theme cookie is set with root / path to keep the theme change between admin and frontend and also on all subdomains for multisite installations.

Because of subdir installation the cookie can't be set on root / path and fails.

I changed the code to write the cookie for both current path and domain and for / path and subdomains to avoid this issue.

Only core update is working currently, for theme files you need to manually update `public/js/admin.js` and `public/themes/landing/js/theme.js` from latest.zip

Thanks for the bug report, if you find other issues let me know.

Best regards
